azure-iot-device/azure/iot/device/__init__.py

```diff
@@ -10,16 +10,17 @@
 from . import iothub
 from . import provisioning
 from . import common
-from . import patch
+from . import patch, patch_documentation
 import sys
 
 if sys.version_info > (3, 5):  # This only works for python 3.5+ at present
     # Dynamically patch the clients to add shim implementations for all the inherited methods.
     # This is necessary to generate accurate online docs.
     # It SHOULD not impact the functionality of the methods themselves in any way.
-    patch.add_shims_for_inherited_methods(IoTHubDeviceClient)  # noqa: F405
-    patch.add_shims_for_inherited_methods(IoTHubModuleClient)  # noqa: F405
-    patch.add_shims_for_inherited_methods(ProvisioningDeviceClient)  # noqa: F405
+    # patch.add_shims_for_inherited_methods(IoTHubDeviceClient)  # noqa: F405
```

Comment:
>patch [](start = 6, length = 5)

Suggest adding a short comment on how to re-generate patch_documentation.py in both places.  Sure, it's easy to do now, but future-us might need some help.